windows-nsis/build-snapshot

```diff
@@ -31,7 +32,7 @@ main() {
 	git clone -q --depth 1 --branch "${OPENVPN_BRANCH}" "${OPENVPN_URL}" || die "openvpn.clone"
 	cd openvpn
 	autoreconf -ivf || die "autoreconf.openvpn"
-	./configure --disable-lzo --disable-snappy --disable-lz4 || die "openvpn.configure"
+	./configure --disable-lzo --disable-lz4 || die "openvpn.configure"
```

Comment:
This makes sense in the "master" branch indeed.

Just so that you know: yesterday I created a "release/2.3" branch in openvpn-build, because the most recent changes to openvpn.nsi were incompatible with OpenVPN 2.3.x. The HEAD of the release/2.3 branch is now at 42aed3b6e1267258 ("Revert "Fail if trying to run I60x installer on Windows XP""), but some of the more recent commits could probably be cherry-picked and applied to release/2.3 too. Feel free to experiment.